openhands/controller/agent_controller.py

```diff
@@ -73,6 +76,8 @@ class AgentController:
         NullObservation,
         ChangeAgentStateAction,
         AgentStateChangedObservation,
+        ReplayCmdRunAction,
```

Comment:
Exclude Replay action and observation from history for now.
TODO: Will need visible and invisible ways of interacting with recordings in the future.